src/network/service.ts

```diff
@@ -38,6 +38,6 @@ export class NetworkService {
 
   async end() {
     await this.network?.remove();
-    await this.logger?.debug("Network Service has been stopped");
+    this.logger.debug("Network Service has been stopped");
```

Comment:
To consider: Places where the core services are starting/stopping, could be reported on the info level so that we as developers see these operations without diving into `debug`. I't like with starting regular apps, things like "starting service" "stopping service" are usually on the INFO level.

Reply:
Good idea